addons/binding/org.openhab.binding.denonmarantz/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,360 @@
+                <label>Zone count of the receiver</label>
+                <description>Number of zones (including main zone).</description>
+                <default>2</default>
+            </parameter>
+
+            <parameter name="host" type="text" required="true" groupName="connectionSettings">
+                <context>network-address</context>
+                <label>AVR Host or IP Address</label>
+                <description>Hostname or IP address of the AVR to control.</description>
+            </parameter>
+
+            <parameter name="telnetEnabled" type="boolean" required="true" groupName="connectionSettings">
+                <label>Use Telnet port?</label>
+                <description>By using telnet the AVR updates are received immediately. Also, some devices only support telnet. However, the AVR only allows 1 simultaneous connection. Uncheck if you are using dedicated apps to control the AVR. Then HTTP polling is used instead.</description>
+                <default>false</default>
+            </parameter>
+
+            <parameter name="telnetPort" type="integer" required="false" groupName="telnetSettings">
+                <label>Telnet port</label>
+                <description>Telnet port used for AVR communication. Normally shouldn't be changed.</description>
+                <default>23</default>
+                <advanced>true</advanced>
+            </parameter>
+
+            <parameter name="httpPort" type="integer" required="true" groupName="httpSettings">
+                <label>HTTP port</label>
+                <description>HTTP Port used for AVR communication. Normally shouldn't be changed.</description>
+                <default>80</default>
+                <advanced>true</advanced>
+            </parameter>
+
+            <parameter name="httpPollingInterval" type="integer" required="true" groupName="httpSettings">
+                <label>Polling interval</label>
+                <description>Refresh interval of the HTTP API in milliseconds</description>
+                <default>5000</default>
+                <advanced>true</advanced>
+            </parameter>
+        </config-description>
+
+    </thing-type>
+
+    <channel-type id="power">
+        <item-type>Switch</item-type>
+        <label>Power</label>
+        <description>Power ON/OFF the AVR</description>
+    </channel-type>
+
+    <channel-type id="mainZonePower">
+        <item-type>Switch</item-type>
+        <label>Power (main zone)</label>
+        <description>Power ON/OFF the main zone of the AVR</description>
+    </channel-type>
+
+    <channel-type id="mainVolume">
+        <item-type>Dimmer</item-type>
+        <label>Main volume</label>
+        <description>Set the volume level of the main zone</description>
+        <category>SoundVolume</category>
+    </channel-type>
+    
+    <channel-type id="mainVolumeDB" advanced="true">
```

Comment:
The original binding supported both, also other AVR bindings support both Dimmer 0-100 style and actual dB value style. Some like OpenHAB to match the exact value the AVR is presenting on the (on-screen) display, this is the dB value. 
Deliberately made these channels 'advanced' to avoid confusion for regular users.